.github/workflows/ci.yml

```diff
@@ -0,0 +1,25 @@
+name: 🧪
+
+on:
+  push:
+  pull_request:
+
+jobs:
+  linters:
+    runs-on: fedora-latest
```

Comment:
This will not work, because GitHub does not provide a VM with such name on the public platform. You can see it for yourself: https://github.com/kpi-web-guild/django-girls-blog-OlenaEfymenko/actions/runs/4521508263/jobs/7963286301?pr=17 — no runner picked up the job in 20 hours. It could be possible if we were to build and integrate our own VMs for testing, additionally to the official ones, but we don't.

Here's the list of available values: https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idruns-on.

Reply:
Fixed to `runs-on: ubuntu-latest`. Marking this conversation as resolved.